validate.py

```diff
@@ -91,55 +87,78 @@ def validate_dates(indi_list, fam_list):
         if not valid_month(birth_date):
             print('Error US42: Invalid birth month for ' +
                   row["NAME"] + ' (' + row["ID"] + ') ')
-        elif not date_compare(birth_date):
-            print('Error US01: Birth date of ' +
-                  row["NAME"] + ' (' + row["ID"] + ') ' +
-                  'occurs after the current date.')
+        else:
+            if not date_compare(birth_date,''):
+                print('Error US01: Birth date of ' +
```

Comment:
Unrelated to this pull-request, but I was trying to think of a way to generalize our prints, they all have a common pattern... Just something to ponder.

Reply:
yes, I tackled that in my homework last week.  I'll try to do some refactoring.